packages/esm-patient-immunizations-app/src/hooks/useImmunizations.ts

```diff
@@ -0,0 +1,22 @@
+import { fhirBaseUrl, openmrsFetch } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import { type FHIRImmunizationBundle } from '../types/fhir-immunization-domain';
+import { mapFromFHIRImmunizationBundle } from '../immunizations/immunization-mapper';
+
+export function useImmunizations(patientUuid: string) {
+  const immunizationsUrl = `${fhirBaseUrl}/Immunization?patient=${patientUuid}`;
+
+  const { data, error, isLoading, isValidating, mutate } = useSWR<{ data: FHIRImmunizationBundle }, Error>(
+    immunizationsUrl,
+    openmrsFetch,
+  );
+  const existingImmunizations = data ? mapFromFHIRImmunizationBundle(data.data) : null;
+
+  return {
+    data: existingImmunizations,
+    isError: error,
```

Comment:
`error` is not a boolean. It's either an Error object or null. `isError` implies that it is a boolean value, which is not correct.

Reply:
Fixed!

Reply:
Oppsss!! CI error, let's get it fixed

Reply:
Done